definitions/ext-flex_remote/definition.yml

```diff
@@ -0,0 +1,32 @@
+domain: EXT
+type: FLEX_REMOTE
+
+synthesis:
+  rules:
+  - identifier: remotehost
+    name: remotehost
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: eventType
+      value: remoteLinuxFSSample
+
+  - identifier: remotehost
+    name: remotehost
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: eventType
+      value: remoteLinuxMemorySample
+
+  - identifier: remotehost
+    name: remotehost
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: eventType
+      value: remoteLinuxOpenFDSample
+
+  dashboardTemplates:
```

Comment:
I am seeing a validation error related to this one:

```
../definitions/ext-flex_remote/definition.yml invalid

[
  {
    "instancePath": "/synthesis",
    "schemaPath": "#/properties/synthesis/additionalProperties",
    "keyword": "additionalProperties",
    "params": {
      "additionalProperty": "dashboardTemplates"
    },
    "message": "must NOT have additional properties"
  }
]
```

Reply:
I think you have extra tabs here, take a look at some existing examples: https://github.com/newrelic/entity-definitions/blob/c83c8348131e5980a2bc6292e18b2c33c9bb6618/definitions/ext-unix_host/definition.yml#L11

Reply:
I checked and there was an error with the indentation. It was fixed